modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -40,6 +40,8 @@ image::ex2-exchange-assets.png[]
 
 == Asset Limits
 
+An asset's version format must use semantic versioning. The maximum length of an asset's name, description, and `versionGroup` is 256 characters, and the maximum combined length of an asset's `groupId`, `assetId`, and `version` is 215 characters. The minimum length of an asset's name, `assetId`, and `versionGroup` is 1 character, and the minimum length of an asset's description is 0 characters.
```

Comment:
Since the second half of the second sentence specifies combined length, is it clear that the first half refers to the length of each?
```suggestion
An asset's version format must use semantic versioning. The maximum length of each of an asset's name, description, and `versionGroup` is 256 characters, and the maximum combined length of an asset's `groupId`, `assetId`, and `version` is 215 characters. The minimum length of an asset's name, `assetId`, and `versionGroup` is 1 character, and the minimum length of an asset's description is 0 characters.
```
Also, I wonder if this would not be more easily presented as a table or list?
```suggestion
Asset Feature.               Maximum Length.            Minimum Length.            Other Requirements
Name
Description
Version
(or?)
`assetId`
'groupId`
`version`
`versionGroup`
```

Reply:
I think a table is better here! Updated.